dist/activate

```diff
@@ -0,0 +1,171 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# set up the environment variables
+KCTF_CONFIG_DIR="${HOME}/.config/kctf"
+KCTF_CTF_DIR="$(realpath --no-symlinks "$(dirname "${BASH_SOURCE-$0}")/..")"
+KCTF_CTF_NAME=$(basename "${KCTF_CTF_DIR}")
+KCTF_BIN="${KCTF_CTF_DIR}/kctf/bin"
+KCTF_SESSION="$(dd if=/dev/urandom bs=1 count=10 2>/dev/null | xxd -ps -c 10)"
+
+function _kctf_download_dependencies {
+  if [[ ! -x "${KCTF_BIN}/yq" ]]; then
+    if [[ -e "${KCTF_BIN}/yq" ]]; then
+      rm "${KCTF_BIN}/yq" >/dev/null
+    fi
+    wget "https://github.com/mikefarah/yq/releases/download/v4.2.0/yq_linux_amd64" -O "${KCTF_BIN}/yq" --quiet || return 1
```

Comment:
maybe this and the hash below can be env vars at the top.

Reply:
done